extensions/binding/org.eclipse.smarthome.binding.tradfri/src/main/java/org/eclipse/smarthome/binding/tradfri/internal/TradfriHandlerFactory.java

```diff
@@ -79,14 +79,14 @@ protected synchronized void removeHandler(ThingHandler thingHandler) {
     private void registerDiscoveryService(TradfriGatewayHandler bridgeHandler) {
         TradfriDiscoveryService discoveryService = new TradfriDiscoveryService(bridgeHandler);
         discoveryService.activate();
-        this.discoveryServiceRegs.put(bridgeHandler.getThing().getUID(), bundleContext
+        this.discoveryServiceRegs.put(bridgeHandler.getThing().getUID(), getBundleContext()
```

Comment:
afair, this has been copied from hue binding and we have similar code in other bindings as well - so it might make sense to adapt them likewise.

Reply:
> so it might make sense to adapt them likewise

As already written above the bundle context member variable of the base class should be made private and the usage should be changed to use the get method. WDYT? What about a transition phase because of that API breakage? Do we need one?